api/tests/unit/application/certificationCourses/certification-course-controller_test.js

```diff
@@ -221,19 +230,24 @@ describe('Unit | Controller | certification-course-controller', function() {
 
     it('should call assessmentRepository#getByCertificationCourseId with request param', () => {
       // when
-      CertificationCourseController.get(request, reply);
+      const promise = CertificationCourseController.get(request, reply);
 
       // then
-      sinon.assert.calledOnce(assessmentRepository.getByCertificationCourseId);
-      sinon.assert.calledWithExactly(assessmentRepository.getByCertificationCourseId, certificationId);
+      return promise.then(() => {
+        sinon.assert.calledOnce(CertificationCourseRepository.get);
+        sinon.assert.calledWithExactly(CertificationCourseRepository.get, certificationId);
+      });
     });
 
     it('should reply the certification course serialized', () => {
       // when
-      CertificationCourseController.get(request, reply);
+      const promise = CertificationCourseController.get(request, reply);
 
       // then
-      sinon.assert.calledWithExactly(reply, certificationSerialized);
+      return promise.then(() => {
+        sinon.assert.calledOnce(reply);
+        sinon.assert.calledWithExactly(reply, certificationSerialized);
```

Comment:
On aurait pu faire un `expect(reply).to.have.been.calledWith(certificationSerialized)`